client/src/components/login/Login.js

```diff
@@ -24,7 +24,6 @@ const FormWrap = styled.div`
 `;
 
 function Login(props) {
-  const [name, setName] = useState('');
```

Comment:
Didn't see where this was being used and didn't like the warnings in the console